dcs/config.py

```diff
@@ -55,5 +55,11 @@ class Config(
 ):
     """Config parameters and their defaults."""
 
-    service_name: str = "dcs"
     api_route: str = "/ga4gh/drs/v1"
+    cache_timeout: int = Field(
```

Comment:
Couldn't we move this parameter into the DataRepositoryConfig? Then we could get rid of the cleanup module. Moreover, the core would not have to depend on the global config.